packages/@okta/i18n/src/properties/login_nl_NL.properties

```diff
@@ -1460,7 +1460,7 @@ oie.browser.error.NotAllowedError = Er is sprake van een time-out voor de bewerk
 
 ## soft_token API errors
 authfactor.challenge.soft_token.invalid_passcode = Je code komt niet overeen met onze gegevens. Probeer het opnieuw.
-authfactor.challenge.soft_token.used_passcode = Elke code kan slechts \u00E9\u00E9n keer worden gebruikt. Wacht even voor een nieuwe code en probeer het opnieuw.
+authfactor.challenge.soft_token.used_passcode = Elke code kan slechts een keer worden gebruikt. Wacht even voor een nieuwe code en probeer het opnieuw.
```

Comment:
unicode to non-unicode change

Reply:
https://www.quora.com/What-is-the-Dutch-equivalent-of-one-Een-or-%C3%A9%C3%A9n

this is a grammatical change but I think either way is fine? (the original one was more explicit on it meaning "one") 

For the ref:
`authfactor.challenge.soft_token.used_passcode = Each code can only be used once. Please wait for a new code and try again.` 

@pengyao-okta  @yasminvanya-okta  can we get a linguist to confirm this is correct? 

In the meantime @nikhilvenkatraman-okta would you be alright if we just left it?

Reply:
I am ok as we try to minimize manual fixes

Reply:
Let me ask linguists.